pachyderm/values.yaml

```diff
@@ -19,16 +19,16 @@ pachd:
     #  memory: 2G  
   logLevel: info
   metrics:
-    enabled: true
+    enabled: "true"
```

Comment:
Can we keep Boolean values as actual booleans in the values file, and turn them into strings if needed by quoting in the templates? That way conditionals are not surprising.

Reply:
Applies to the other instances of true/false throughout.